### PR TITLE
[Fix](schema scanner) Fixed the problem of overflow when multiplying two INT

### PR DESCRIPTION
**Pull request by Cai-Yao**
# Proposed changes

Issue Number: close #xxx

### Before

```
_desc_result.columns[i].columnDesc.columnLength
```

### After

```
static_cast<int64_t>(_desc_result.columns[i].columnDesc.columnLength) * 4
```

avoid 2147483643 * 4 = -20

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/exec/schema_scanner/schema_columns_scanner.cpp [hunk 1]
```diff
@@ -455,7 +455,9 @@ Status SchemaColumnsScanner::_fill_block_impl(vectorized::Block* block) {
             if (data_type == TPrimitiveType::VARCHAR || data_type == TPrimitiveType::CHAR ||
                 data_type == TPrimitiveType::STRING) {
                 if (_desc_result.columns[i].columnDesc.__isset.columnLength) {
-                    srcs[i] = _desc_result.columns[i].columnDesc.columnLength * 4;
+                    srcs[i] =
+                            static_cast<int64_t>(_desc_result.columns[i].columnDesc.columnLength) *
+                            4;
```
Comment (zhangstar333) [inline be/src/exec/schema_scanner/schema_columns_scanner.cpp:460]: seems could use `length * 4L`

Comment (Cai-Yao) [reply be/src/exec/schema_scanner/schema_columns_scanner.cpp]: Done

**Review by zhangstar333 — APPROVED** [review]

**Comment by zhangstar333** [comment]
run buildall

**Comment by Cai-Yao** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**